ext/strings_test.go

```diff
@@ -105,6 +105,20 @@ var stringTests = []struct {
 	{expr: `['x', 'y'].join('-') == 'x-y'`},
 	{expr: `[].join() == ''`},
 	{expr: `[].join('-') == ''`},
+	// Escaping tests.
```

Comment:
This looks right to me, but if you want to be extra sure, you could implement "unquote" and then add a fuzzer test. The fuzzer test could run fuzzed inputs through a quote->unquote to make sure they round trip. I've had good luck finding subtle bugs with this on other escaping utilities.

Reply:
Tests that roundtrip in both directions, quoted <-> unquoted, would probably be useful. 

It's also useful to look at src/strconv/quote.go to determine how to handle unprintable characters which are written as unicode escape sequences. I'm not sure we want to support everything that is supported by that method and whether the question is whether we're only handling quoting only Ascii or also all of unicode?

Can you add tests which include quotes mid-string, e.g. `strings.quote('quote"int(42)"')`

Reply:
(forgot to mark this as resolved - resolving)